app/controllers/admin/feature_flags_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Admin::FeatureFlagsController < ApplicationController
+  before_action :set_tenant
+
+  def index
+    authorize([:admin, :feature_flag])
+    @feature_flags = @tenant.list_features
+    @enabled_features = @tenant.feature_flags
+  end
+
+  def update
+    authorize([:admin, :feature_flag])
+    @tenant.feature_flags = feature_flags_params[:enabled] == "true" ? @tenant.feature_flags.union([params[:id]]) : @tenant.feature_flags - [params[:id]]
```

Comment:
Tento zapis neviem uplne precitat na prvu lebo nepouzivas uplne bezne metody. Ak chceme len pridat feature flag ak je enabled tak by som mozno zvazil takyto zapis.

```suggestion
    if feature_flags_params[:enabled] == "true"
      @tenant.feature_flags << params[:id] 
    else
      @tenant.feature_flags.delete(params[:id])
    end
```